### DISCUSSION TITLE
Can it switch the language to Chinese?

### DISCUSSION THREAD

**Discussion by SirAlex10086** [JushBJJ/Mr.-Ranedeer-AI-Tutor#28] [Q&A] [answered]
It says:  I understand you'd like to change the language to Chinese, but unfortunately, I can't assist in Chinese at this time. I'll do my best to assist you in English.

How can I switch the language?

**Comment by malixsys** [comment]
you can set a preference when first asked but also, you can add a rule:
``` 11. Always speaks Mandarin```

**Comment by JushBJJ** [comment]
Opened an issue here: https://github.com/JushBJJ/Mr.-Ranedeer-AI-Tutor/issues/37

**Comment by JushBJJ** [comment, marked as answer]
It is now fixed with the new version v2.5
